mtest/importmidi/tst_importmidi.cpp

```diff
@@ -20,20 +20,20 @@
 #include "libmscore/chord.h"
 #include "libmscore/note.h"
 #include "libmscore/keysig.h"
-#include "mscore/exportmidi.h"
+#include "importexport/midiexport/exportmidi.h"
```

Comment:
This file cannot be found. Instead, try
```
#include "audio/exports/exportmidi.h"
```

Reply:
Thank you.
In these days, MuseScore files are continuosly changing place in the source code.